sa/_db/migrations/20150922165824_UpdateSerialNumbersOfOldCerts.sql

```diff
@@ -0,0 +1,12 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+update certificates set serial = concat('0000', serial);
```

Comment:
Our style so far in SQL has been to capitalize the keywords.

Also, please add a brief comment explaining the motivation and linking to the issue.